test/onnx/test_custom_ops.py

```diff
@@ -54,6 +54,50 @@ def symbolic_custom_add(g, self, other):
         caffe2_out = prepared.run(inputs=[x.cpu().numpy(), y.cpu().numpy()])
         np.testing.assert_array_equal(caffe2_out[0], model(x, y).cpu().numpy())
 
+    def test_symbolic_error(self):
+        op_source = """
+        #include <torch/script.h>
+
+        torch::Tensor custom_add(torch::Tensor self, torch::Tensor other) {
+          return self + other;
+        }
+
+        static auto registry =
+          torch::RegisterOperators("custom_namespace::custom_add", &custom_add);
+        """
+
+        torch.utils.cpp_extension.load_inline(
+            name="custom_add",
+            cpp_sources=op_source,
+            is_python_module=False,
+            verbose=True,
+        )
+
+        class CustomAddModel(torch.nn.Module):
+            def forward(self, a, b):
+                return torch.ops.custom_namespace.custom_add(a, b)
+
+        graph_holder = []
+
+        def symbolic_custom_add(g, self, other):
+            graph_holder.append(g)
+            return g.op("Add", self, other)
+
+        from torch.onnx import register_custom_op_symbolic
```

Comment:
nit: inline imports

Reply:
updated, turned out the test is identical to the existing one, just keeping the comments.